utils/common.c

```diff
@@ -86,6 +95,21 @@ glusterBlockParseSize(const char *dom, char *value)
     /* fall through */
   case 'b':
   case '\0':
+    if (sizef < GB_DEFAULT_SECTOR_SIZE) {
+      MSG("minimum acceptable block size is %d bytes\n", GB_DEFAULT_SECTOR_SIZE);
+      LOG(dom, GB_LOG_ERROR, "minimum acceptable block size is %d bytes",
+          GB_DEFAULT_SECTOR_SIZE);
+      return -1;
+    }
+
+    if (sizef % GB_DEFAULT_SECTOR_SIZE) {
```

Comment:
@pranithk do you foresee any issue with continuing here with a msg ?

Reply:
@pkalever @lxbsz Do you see any value in allowing a size that is going to be rounded down?

Reply:
@pranithk Do you mean just fail it here when the size is not aligned ?

Reply:
@lxbsz Yes, if there is no value in creating blocks that are not aligned, I don't see any reason to allow it. You can let me know if you know of any cases where this is useful, otherwise failing it is better with a helpful message saying the block needs to be aligned with the $sector_size

Reply:
@pranithk what will happen if we allow un-aligned sizes and we round down it in the code(ourself), rather restricting at the user level ? At *max* we might waste (sector_size - 1) space ? Are you concerned about this space ?

Reply:
@pkalever Please answer my question in the past 2 comments... what is the value in allowing a size that is rounded down. Once I understand the reasoning I will be able to appreciate this approach better.

Reply:
@pkalever @lxbsz Any updates on this?

Reply:
@pranithk From my side, I couldn't foresee the use case for the not aligned block device. So just fail it should be a better choice.